<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1060</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(197, 198, 196);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="entryGroup">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>1201</width>
      <height>571</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="entrance_table_title">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>593</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
border-radius: 10px;
background-color: #ABA49B;
color:white;

</string>
     </property>
     <property name="text">
      <string>RECENTLY ENTERED VEHICLES</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="CustomTableWidget" name="entrance_table">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>1171</width>
       <height>521</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1248</width>
       <height>900</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget{
border:2px solid black;
border-radius:10px;
padding:2px;
background-color:#E6DFDF;
}
QHeaderView::section {
background-color: rgba(171,164,155,0.8);
}
</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>240</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>60</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
    </widget>
    <zorder>entrance_table</zorder>
    <zorder>entrance_table_title</zorder>
   </widget>
   <widget class="QGroupBox" name="exitGroup">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>650</y>
      <width>1201</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="CustomTableWidget" name="exit_table">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>1131</width>
       <height>341</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1248</width>
       <height>900</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget{
border:2px solid black;
border-radius:10px;
padding:2px;
background-color:rgba(230,223,223, 0.5);
}

QHeaderView::section {
background-color: rgba(171,164,155,0.8)
}
</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>240</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>60</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
    </widget>
    <widget class="QLabel" name="exit_table_title">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>0</y>
       <width>563</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
border-radius: 10px;
background-color: #ABA49B;
color:white;

</string>
     </property>
     <property name="text">
      <string>Recent Exit</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="rfid">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>10</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsButton">
    <property name="geometry">
     <rect>
      <x>1840</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../static/settings_icon.png</normaloff>../static/settings_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QStackedWidget" name="coveredParkingGroup">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>90</y>
      <width>620</width>
      <height>510</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>620</width>
      <height>510</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QStackedWidget{background-color: rgba(230,223,223,1);
border:2px solid black;
border-radius:10px;
padding:2px;}</string>
    </property>
    <widget class="QWidget" name="coveredParkingGroupPage1">
     <property name="styleSheet">
      <string notr="true">background-color: rgba(230,223,223,1);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="coveredParkingMainVertical">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="topMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="available_space_title">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>620</y>
      <width>430</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;
background-color: #ABA49B;

</string>
    </property>
    <property name="text">
     <string>Available Parking Spaces</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QStackedWidget" name="availableParkingSpaceStackedWidget">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>659</y>
      <width>590</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QStackedWidget{background-color: rgba(230,223,223,1);
border:2px solid black;
border-radius:10px;
padding:2px;}
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="parking">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="parkingLayoutFrame">
          <property name="minimumSize">
           <size>
            <width>580</width>
            <height>350</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgba(230,223,223,1);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="location">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="locationImage">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="covered_parking_title">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>50</y>
      <width>430</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;
background-color: #ABA49B;

</string>
    </property>
    <property name="text">
     <string>Covered Parking Status</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="announcementLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>1201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#dc3545;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>entryGroup</zorder>
   <zorder>exitGroup</zorder>
   <zorder>rfid</zorder>
   <zorder>settingsButton</zorder>
   <zorder>coveredParkingGroup</zorder>
   <zorder>availableParkingSpaceStackedWidget</zorder>
   <zorder>available_space_title</zorder>
   <zorder>covered_parking_title</zorder>
   <zorder>announcementLabel</zorder>
  </widget>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTableWidget</class>
   <extends>QTableWidget</extends>
   <header>widgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
